src/ts/components/subtitleoverlay.ts

```diff
@@ -73,6 +137,193 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     // Init
     subtitleClearHandler();
   }
+
+  /**
+   * updateSubtitleOverlayConfig updates style for the subtitle overlay based on init config
```

Comment:
Remove (wrong) method name from documentation. The documentation is attached to the function with the name, so it's an unnecessary redundancy to add it to the docs. Also please rephase to a real sentence.